collector/tcpstat_linux.go

```diff
@@ -127,7 +127,7 @@ func parseTCPStats(r io.Reader) (map[tcpConnectionState]float64, error) {
 			continue
 		}
 		if len(parts) < 5 {
-			return nil, fmt.Errorf("invalid TCP stats line: %q", line)
+			return nil, fmt.Errorf("invalid TCP stats line: %w", line)
```

Comment:
That makes ci fail: This isn't an error.

Reply:
Oops!